inscriptions/ISic1470.xml

```diff
@@ -169,35 +174,26 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR108697" xml:lang="grc">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="VM"><!-- check line division -->
```

Comment:
Ha, yes, the line-division is a challenge, given that it's a multi-directional text. I think the first EDR version is better than the second, but really it's about counting a change of direction as a line-break in this case... I don't see how one can say anything about the position / direction of the final missing sigma in any case. This is fine, but I'd make 4a into 5, move the final sigma up onto line 5, and not have a line 6. M-P's text is not wonderfully clear in this regard. NB I'll make the change after merging the pull, don't worry about it.